examples/decoding/plot_decoding_xdawn_eeg.py

```diff
@@ -9,14 +9,15 @@
 
 References
 ----------
-[1] Rivet, B., Souloumiac, A., Attina, V., & Gibert, G. (2009). xDAWN
-algorithm to enhance evoked potentials: application to brain-computer
-interface. Biomedical Engineering, IEEE Transactions on, 56(8), 2035-2043.
-
-[2] Rivet, B., Cecotti, H., Souloumiac, A., Maby, E., & Mattout, J. (2011,
-August). Theoretical analysis of xDAWN algorithm: application to an
-efficient sensor selection in a P300 BCI. In Signal Processing Conference,
-2011 19th European (pp. 1382-1386). IEEE.
+
```

Comment:
ok - just seems like we have inconsistency in this now, e.g.,  [here](https://github.com/mne-tools/mne-python/blob/master/mne/decoding/time_gen.py#L1016) there is no space. Is there a space here because it's an example as opposed to a docstring?